tests/saver/test_save_model_results.py

```diff
@@ -0,0 +1,201 @@
+import pandas as pd
+import pytest
+
+import save_results._save_results
+
+from cascade.core.context import ExecutionContext
+import cascade.saver
+import cascade.saver.save_model_results as smr
+from cascade.dismod.db.wrapper import _get_engine, DismodFile
+
+DRAWS_INPUT_FILE_PATTERN = "all_draws.h5"
+
+
+@pytest.fixture(scope="module")
+def predict_df():
```

Comment:
You have a fixture for this in both test files. You can move this into a module level conftest.py if it makes sense for them to share.